environments/.caas/hooks/post.yml

```diff
@@ -0,0 +1,86 @@
+- name: Persist login hostkey across rebuilds
+# Need NFS for this so can't do it before the appliance plays
+  hosts: login
+  gather_facts: no
+  become: yes
+  roles:
+    - persist_hostkeys
+
+# Configure the Zenith clients that are required
+# First, ensure that podman is installed on all hosts that will run Zenith clients
+- hosts: zenith,!podman
+  tasks:
+    - import_role:
+        name: podman
+        tasks_from: prereqs.yml
+    - import_role:
+        name: podman
+        tasks_from: config.yml
+
+- hosts: grafana
+  tasks:
+    - name: Deploy the Zenith client for Grafana
+      include_role:
+        name: zenith_proxy
+      vars:
+        zenith_proxy_service_name: zenith-monitoring
+        # Use the IP address for the upstream host
+        zenith_proxy_upstream_host: "{{ ansible_host }}" # IP
+        zenith_proxy_upstream_port: "{{ grafana_port }}"
+        zenith_proxy_client_token: "{{ zenith_token_monitoring }}"
+        zenith_proxy_client_auth_params:
+          tenancy-id: "{{ openstack_project_id }}"
+        zenith_proxy_mitm_enabled: yes
+        zenith_proxy_mitm_auth_inject: basic
+        zenith_proxy_mitm_auth_basic_username: "{{ grafana_security.admin_user }}"
+        zenith_proxy_mitm_auth_basic_password: "{{ grafana_security.admin_password }}"
+      when: zenith_subdomain_monitoring is defined
+
+- hosts: openondemand
+  tasks:
+    - name: Deploy the Zenith client for OOD
+      include_role:
+        name: zenith_proxy
+      vars:
+        zenith_proxy_service_name: zenith-ood
+        # Use the IP address for the upstream host
+        zenith_proxy_upstream_scheme: https
+        zenith_proxy_upstream_host: "{{ ansible_host }}" # IP
+        zenith_proxy_upstream_port: 443
+        zenith_proxy_client_token: "{{ zenith_token_ood }}"
+        zenith_proxy_client_auth_params:
+          tenancy-id: "{{ openstack_project_id }}"
+        zenith_proxy_mitm_enabled: yes
+        zenith_proxy_mitm_auth_inject: basic
+        zenith_proxy_mitm_auth_basic_username: azimuth
+        zenith_proxy_mitm_auth_basic_password: "{{ vault_azimuth_user_password }}"
+      when: zenith_subdomain_ood is defined
+
+# Run hpctests if set in UI
+- hosts: hpctests[0]
+  become: false
+  gather_facts: false
+  tasks:
+    - import_role:
+        name: hpctests
+      when: cluster_run_validation | default(false) | bool
+
+# Write the outputs as the final task
+- hosts: localhost
+  tasks:
+    - debug: var=outputs
+      vars:
+        # Ansible has a fit when there are two 'hostvars' evaluations in a resolution chain,
+        # so we have to repeat logic here unfortunately
+        outputs: >-
+          {{-
+            { "cluster_access_ip": hostvars[groups['openstack'][0]].cluster_gateway_ip } |
```

Comment:
Does the "Details" panel in Azimuth render with correct information still?

Reply:
Yes